tests/common.h

```diff
@@ -0,0 +1,93 @@
+#ifndef COMMON_H
+#define COMMON_H
+
+enum crustls_demo_result
+{
+  CRUSTLS_DEMO_OK,
+  CRUSTLS_DEMO_ERROR,
+  CRUSTLS_DEMO_AGAIN,
+  CRUSTLS_DEMO_EOF,
+  CRUSTLS_DEMO_CLOSE_NOTIFY,
+};
+
+/* A growable vector of bytes. */
+struct bytevec {
+  char *data;
+  size_t len;
+  size_t capacity;
+};
+
+struct conndata_t {
```

Comment:
nit
```suggestion
struct conndata {
```
to match struct naming convention in the rest of the project.